rerun_py/rerun_sdk/rerun/_rerun2/datatypes/transform3d_ext.py

```diff
@@ -183,6 +95,15 @@ def _optional_translation_to_arrow(translation: Vec3D | None) -> pa.Array:
         return pa.FixedSizeListArray.from_arrays(translation.xyz, type=Vec3DType().storage_type)
 
 
+def _optional_rotation_to_arrow(rotation: Rotation3D | None, storage_type: pa.DataType) -> pa.Array:
```

Comment:
Maybe we should just codegen `Rotation3DArray._optional_native_to_pa_array`?

Reply:
hmm worth considering, do we have it in a lot of other places too?